mediacloud/api.py

```diff
@@ -94,7 +94,7 @@ def source_list(self, platform: Optional[str] = None, name: Optional[str] = None
     def feed_list(self, source_id: Optional[int] = None,
                   modified_since: Optional[Union[dt.datetime, int, float]] = None,
                   modified_before: Optional[Union[dt.datetime, int, float]] = None,
-                  limit: Optional[int] = 0, offset: Optional[int] = 0) -> Dict:
+                  limit: Optional[int] = 0, offset: Optional[int] = 0, return_details=False) -> Dict:
```

Comment:
Since everything is (over!) typed, I would do `return_details: bool = False`

`Optional[thing]` == `Union[thing, None]` == `thing | None`
which explicitly means that `None` is a valid value to pass (and often means a default value will be supplied if you pass an explicit `None`), which, for `limit` and `offset` seems excessive, since their default values are NOT `None`, but I think changing them is probably excessive (unless passing `None` didn't work at all).

Reply:
Good catch, thanks!